src/WPF/ArcGISRuntime.WPF.Samples/Samples/Data/EditAndSyncFeatures/EditAndSyncFeatures.xaml.cs

```diff
@@ -0,0 +1,511 @@
+﻿// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using ArcGISRuntime.Samples.Managers;
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.Tasks;
+using Esri.ArcGISRuntime.Tasks.Offline;
+using Esri.ArcGISRuntime.UI;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Media;
+
+namespace ArcGISRuntime.WPF.Samples.EditAndSyncFeatures
+{
+    public partial class EditAndSyncFeatures
+    {
+        // Enumeration to track which phase of the workflow the sample is in
+        private enum EditState
+        {
+            NotReady, // Geodatabase has not yet been generated
+            Editing, // A feature is in the process of being moved
+            Ready // The geodatabase is ready for synchronization or further edits
+        }
+
+        // URI for a feature service that supports geodatabase generation
+        private Uri _featureServiceUri = new Uri("https://sampleserver6.arcgisonline.com/arcgis/rest/services/Sync/WildfireSync/FeatureServer");
+
+        // Path to the geodatabase file on disk
+        private string _gdbPath;
+
+        // Task to be used for generating the geodatabase
+        private GeodatabaseSyncTask _gdbSyncTask;
+
+        // Flag to indicate which stage of the edit process we're in
+        private EditState _readyForEdits = EditState.NotReady;
+
+        // Hold a reference to the generated geodatabase
+        private Geodatabase _resultGdb;
+
+        public EditAndSyncFeatures()
+        {
+            InitializeComponent();
+
+            // Create the UI, setup the control references and execute initialization
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create a tile cache and load it with the SanFrancisco streets tpk
+            TileCache _tileCache = new TileCache(await GetTpkPath());
```

Comment:
Variables that begin with an underscore ( _XXXXX ) should be declared as global in scope. In this case you have a local variable and it should be names something like: myToeCache not _tileCache.